malariagen_data/anoph/het_params.py

```diff
@@ -58,6 +58,11 @@
     "Passed through to bokeh circle() function.",
 ]
 
+line_kwargs: TypeAlias = Annotated[
+    Mapping,
+    "Passed through to bokeh line() function.",
+]
+
```

Comment:
Out of scope for this PR?

Reply:
Used in methods like `plot_cnv_hmm_coverage_track()`.